src/app/accounts/accounts/accounts.component.html

```diff
@@ -1,16 +1,15 @@
-<div class="accounts">
-  <mat-nav-list class="accounts__add-account drawer-menu">
-    <mat-list-item (click)="handleAddAccountClick()">
-      <mat-icon matListIcon>add</mat-icon>
-      Add new account
-    </mat-list-item>
-  </mat-nav-list>
-  <mat-list class="accounts__menu drawer-menu">
-    <mat-list-item *ngFor="let name of accounts" class="accounts__item drawer-menu__item">
-      <mat-icon matListIcon>account_balance</mat-icon>
-      {{name}}
-      <app-button-more [position]="'after'" class="accounts__button-more drawer-menu__button-more"
-        (editClick)="editAccount()" (deleteClick)="deleteAccount()"></app-button-more>
-    </mat-list-item>
-  </mat-list>
-</div>
\ No newline at end of file
+<mat-nav-list class="drawer-menu">
+  <mat-list-item (click)="handleAddAccountClick()">
+    <mat-icon matListIcon>add</mat-icon>
+    Add new account
+  </mat-list-item>
+</mat-nav-list>
+<mat-list class="drawer-menu">
+  <mat-list-item *ngFor="let account of accounts" class="drawer-menu__item">
+    <mat-icon matListIcon>account_balance</mat-icon>
+    {{account}}
+    <app-button-more [position]="'after'" [data]="{type: 'account', name: account}" class="drawer-menu__button-more"
```

Comment:
why are you writing it like `[position]="'after'"`. Could it be just `position="after"` ?